src/components/style/form.css

```diff
@@ -0,0 +1,44 @@
+.form-section {
```

Comment:
Good task for improving the color, however, kindly pay attention to original design. Kindly ensure that:
- [ ] Font matches the design template
- [ ] The logo _Bookstore CMS_  have  bluish color (see the actual color used in Zeplin)
- [ ] _Comments_ , _Remove_ and _Edit_  have sea blue color (see the actual color used in Zeplin), and do not have blue background
- [ ] Add a blue ring by the stats in the middle
- [ ] Spacing should match [given template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
- [ ] Add the section heading, _ADD NEW BOOK_, before add new book form fields

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/185771469-df94abc7-e2dc-4bf5-be41-7cb9287de4b7.png)


**Original design:**
![image](https://user-images.githubusercontent.com/12745474/185769922-d8cda7d5-8f2f-420b-ac2c-26431dc3b23c.png)